server/database/fakeData/index.ts

```diff
@@ -0,0 +1,48 @@
+import sequelize from '../config/connection';
+// eslint-disable-next-line import/extensions
+import * as data from './data.json';
+
+import {
+  Campaign,
+  Donor,
+  Family,
+  Donation,
+  Contact,
+  Category,
+  Capon,
+} from '../models';
+
+const {
+  campaigns, capons, categories, messages, donations, donors, families,
+} = data;
```

Comment:
If you are declaring several variables then don't put them in one long line like this, rewrite them like this
```js
const {
campaigns, 
capons, 
categories, 
messages, 
donations, 
donors, 
families,
} = data;
```